csharp-advanced-topics/ActionInCSharp/ActionInCSharp/Program.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+
+namespace ActionInCSharp
+{
+    class Program
+    {
+        private static int resultNumber = 0;
+        private static string resultMessage = "";
+        static void Main(string[] args)
```

Comment:
Newline needed

Reply:
I've created branch from my main branch named as "naman/ActionAndFunc" but cannot make commit and push it gives me.
![image](https://user-images.githubusercontent.com/66085031/155975508-43089fa5-ab15-4381-a5a9-82e603a33b8b.png)

Now as I want to commit the changes to this new branch and then make a PR to your main branch.

Reply:
You're trying to push directly to our repository. You need to use your own fork, and create a PR from that fork to our repo.